Description: Fix new test cases, install data file to meet need of multi-arch
 Copy nedded dict files to ${PROJECT_BINARY_DIR}/data for tests.
 Install data files to ${DIR_LIBRARY} instead of ${DIR_SHARE}
Author: Kan-Ru Chen <koster@debian.org>, YunQiang Su <wzssyqa@gmail.com>

Index: opencc/data/CMakeLists.txt
===================================================================
--- opencc.orig/data/CMakeLists.txt	2012-03-29 01:56:20.043932047 +0800
+++ opencc/data/CMakeLists.txt	2012-03-29 03:16:12.183694991 +0800
@@ -92,4 +92,10 @@
 		${CHINA_DICT_FILES}
 	DESTINATION
 		${DIR_SHARE_OPENCC}
-)
\ No newline at end of file
+)
+
+if (BUILD_TESTING)
+	foreach (DICT ${TAIWAN_DICT_FILES} ${CHINA_DICT_FILES})
+		configure_file(${DICT} ${PROJECT_BINARY_DIR}/data COPYONLY)
+	endforeach (DICT)
+endif (BUILD_TESTING)
Index: opencc/CMakeLists.txt
===================================================================
--- opencc.orig/CMakeLists.txt	2012-03-29 01:56:20.043932047 +0800
+++ opencc/CMakeLists.txt	2012-03-29 03:16:34.919807728 +0800
@@ -20,6 +20,7 @@
 cmake_minimum_required(VERSION 2.8)
 set (PACKAGE_NAME opencc)
 project (${PACKAGE_NAME} C)
+include (CTest)
 enable_testing()
 
 ######## Package information
@@ -86,7 +87,7 @@
 	set (DIR_ETC ${SYSCONF_INSTALL_DIR})
 endif (DEFINED SYSCONF_INSTALL_DIR)
 
-set (DIR_SHARE_OPENCC ${DIR_SHARE}/opencc)
+set (DIR_SHARE_OPENCC ${DIR_LIBRARY}/opencc)
 set (DIR_SHARE_LOCALE ${DIR_SHARE}/locale)
 
 ######## Configuration
